Build/ProjectTypes/Analyzer.props

```diff
@@ -0,0 +1,24 @@
+﻿<Project>
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <LangVersion>9</LangVersion>
```

Comment:
We don't do this often, but here a comment that LangVersion=9 is needed for netstandard2.0 compatibility would be great. Otherwise, we start guessing when we visit the file the next time.